Civi/Funding/Controller/DrawdownDocumentDownloadController.php

```diff
@@ -62,24 +62,26 @@ public function handle(Request $request): Response {
    * @throws \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
    */
   private function download(int $drawdownId): Response {
-    if (NULL === $this->drawdownManager->get($drawdownId)) {
+    $drawdown = $this->drawdownManager->get($drawdownId);
+    if (NULL === $drawdown) {
       throw new AccessDeniedHttpException();
```

Comment:
Yes, theoretically it could be an ID that doesn't exist. Though we cannot differentiate here and I don't think it is necessary. A link with an non-existing ID shouldn't be created, anyway.